src/core/main.cpp

```diff
@@ -137,6 +142,14 @@ int main( int argc, char * * argv )
 		}
 	}
 
+#ifndef LMMS_WIN32
```

Comment:
Whoops... typo here... should be `LMMS_BUILD_WIN32`.  Travis is pissed off at me now.